ruma-events/src/room/member.rs

```diff
@@ -406,8 +409,8 @@ mod tests {
                 && third_party_displayname == "alice"
                 && mxid == "@alice:example.org"
                 && signatures == btreemap! {
-                    "magic.forest".to_owned() => btreemap! {
-                        "ed25519:3".to_owned() => "foobar".to_owned()
+                    "magic.forest".try_into().unwrap() => btreemap! {
+                        "ed25519:3".try_into().unwrap() => "foobar".to_owned()
```

Comment:
You can use the identifier macros for these tests (`ruma_identifiers::{server_key_id, server_name}`).